src/providers/oracle/qgsoraclesourceselect.h

```diff
@@ -134,7 +134,7 @@ class QgsOracleSourceSelect : public QDialog, private Ui::QgsDbSourceSelectBase
     //!Sets a new regular expression to the model
     void setSearchExpression( const QString& regexp );
 
-    void on_buttonBox_helpRequested() { QgsContextHelp::run( metaObject()->className() ); }
+    void on_buttonBox_helpRequested() { QgsHelp::openHelp( QStringLiteral( "working_with_vector/supported_data.html#id2" ) ); }
```

Comment:
as @m-kuhn already said, this does not affect way we use to write docs. If you change something in the documentation, e.g. anchor name or move some section to another place you will need to update corresponding link in the code too. But I agree with Matthias here, that such changes should not be too frequent, at least within single version like 3.x, 4.x and so on